storage/innobase/srv/srv0start.cc

```diff
@@ -205,18 +205,12 @@ mysql_pfs_key_t srv_ts_alter_encrypt_thread_key;
 mysql_pfs_key_t parallel_rseg_init_thread_key;
 #endif /* UNIV_PFS_THREAD */
 
-<<<<<<< HEAD
 int unlock_keyrings(THD *thd);
 
 #ifdef WITH_WSREP
 extern bool wsrep_recovery;
 #endif /* WITH_WSREP */
 
-||||||| merged common ancestors
-int unlock_keyrings(THD *thd);
```

Comment:
If PS has removed this function, then it must be removed in PXC as well.

Reply:
Done.